javascript-carousel/main.js

```diff
@@ -0,0 +1,59 @@
+let counter = 0;
+const $allImages = document.querySelectorAll('img');
+const $allDots = document.querySelectorAll('i.fa-circle');
+const dot1 = document.querySelector('i[position="1"]');
+const dot2 = document.querySelector('i[position="2"]');
+const dot3 = document.querySelector('i[position="3"]');
+const dot4 = document.querySelector('i[position="4"]');
+const dot5 = document.querySelector('i[position="5"]');
```

Comment:
Try to rewrite your code so that you could add a dot and an image to the HTML but you wouldn't have to change your  JavaScript.